docs/_config.yml

```diff
@@ -17,18 +17,20 @@ remote_theme: pmarsceill/just-the-docs@v0.4.1
 
 # Aux links for the upper right navigation
 aux_links:
-  'Docs':
+  'Home':
     - 'https://docs.lakefs.io/'
   'Blog':
     - 'https://lakefs.io/blog/'
-  'Community':
-    - 'https://lakefs.io/community/'
```

Comment:
Why are we replacing it? Community is not just Slack alone. @adipolak WDYT?

Reply:
agree.

Reply:
Yes, let's keep it as the community page please. I think it's a good page - it doesn't have any marketing 'smells' if you see what I mean :-) 
(well if I was being really fussy I'd remove webinars from `events`, but that's all ;-) )

Reply:
Okay, no problem.

Reply:
I changed it to the right link :-)